docs/refs.bib

```diff
@@ -133,6 +133,19 @@ @article{Cabello_2002_NParticle
   url={https://arxiv.org/abs/quant-ph/0203119}
 }
 
+
+@misc{Campos_2024_AntiDistinguishability,
```

Comment:
The first word of the title key is the first noun in the title. In this case it would be "epistemic" so

```suggestion
@misc{Campos_2024_Epistemic,
```